xml/http_servers.xml

```diff
@@ -1907,6 +1907,27 @@
       <param pos="1" name="service.version"/>
    </fingerprint>
 
+   <fingerprint pattern="^BBC (\d+\.\d+\.\d+); ovbbccb \d+\.\d+\.\d+$">
+      <example>BBC 06.00.083; ovbbccb 06.00.083</example>
+      <example>BBC 06.20.050; ovbbccb 06.20.050</example>
+      <example>BBC 11.10.035; ovbbccb 11.10.035</example>
+      <description>OpenView ovbbccb </description>
+      <param pos="0" name="service.vendor" value="HP"/>
+      <param pos="0" name="service.product" value="OpenView ovbbccb"/>
```

Comment:
Using just two examples:

```
<example>BBC 06.20.101; ovbbccb 06.20.101</example>
<example>BBC 05.20.050; coda 10.00.123</example>
```

Based on what you've said, I think we should have one fingerprint per component (CB, CODA, LLB, RCP), and for each we'd assert:

```
<param pos="0" name="service.vendor" value="HP"/>
<param pos="0" name="service.family" value="OpenView"/>
<param pos="0" name="service.component.vendor" value="HP"/>
<param pos="0" name="service.component.family" value="OpenView"/>
<param pos="0" name="service.component.product" value="<That component name>"/>
<param pos="1" name="service.component.version"/>
```